db/migrate/20210205133627_remove_encrypted_password_from_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveEncryptedPasswordFromUser < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :users, :encrypted_password, type: :string
```

Comment:
`add_column :users, :cleartext_password, type: :string`

Reply:
What?

Reply:
Surely you mean `plaintext` 😂

Reply:
What do we replace encrypted password with?

Reply:
Sorry, this was a joke, didn't mean to confuse things

Reply:
Cool cool